learningpath-api/src/main/resources/no/ndla/learningpathapi/db/migration/V28__AddLastSharedFieldToFolders.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE folders
+ADD COLUMN last_shared TIMESTAMP DEFAULT NULL;
```

Comment:
Slå sammen disse to migreringene. Dette er en endring og ikkje to, spesielt siden du endrer på samme tabell.
Du har ikkje brukt last_updated så då treng du heller ikkje last_shared.
```suggestion
ADD COLUMN shared TIMESTAMP DEFAULT NULL;
```